samples/server/petstore/php-slim/phpunit.xml.dist

```diff
@@ -19,8 +19,8 @@
     </testsuites>
     <filter>
         <whitelist processUncoveredFilesFromWhitelist="true">
-            <directory suffix=".php">./lib/Api</directory>
-            <directory suffix=".php">./lib/Model</directory>
+            <directory suffix=".php">./lib//Api</directory>
+            <directory suffix=".php">./lib//Model</directory>
```

Comment:
@ybelenko I believe the double forward slash won't cause any issue. Please correct me if I'm wrong.

Reply:
Hmmm, and what are benefits from double forward slash, may I ask? :smile:

Reply:
Well I do not know. I spotted the double forward slash and just asking to make sure phpunit still works fine with it.

Reply:
Just checked on Win8.1, phpunit works fine. These slash changes is result of #610  . I just use `{{apiSrcPath}}` and `{{modelSrcPath}}` which defined in AbstractPhpCodegen.

Reply:
👌